src/main/resources/templates/interest/form.html

```diff
@@ -45,28 +45,35 @@ <h4>Visibility</h4>
             </section>
 
             <section class="input-param">
-                <h4>Criteria</h4>
+                <h4>Rating criteria</h4>
                 <div id="criteriaContainer" class="criteria-container">
-                    <label th:if="*{id == null}">
+                    <label th:if="*{id == null}" class="list-record first-criterion">
                         <input class="input-text-field" type="text" name="criteriaNames" required/>
                     </label>
                     <th:block th:if="*{id != null}">
-                        <label>
+                        <label class="list-record first-criterion">
                             <input class="input-text-field" type="text" name="criteriaNames"
                                    th:value="${interest.criteria[0]?.name}" required/>
                         </label>
 
                         <div th:each="criterion, iterStat : ${interest.criteria}" th:if="${iterStat.index > 0}">
-                            <label class="criterion">
+                            <label class="list-record">
                                 <input class="input-text-field" type="text" name="criteriaNames"
                                        th:value="${criterion.name}" required/>
-                                <button class="delete-button" type="button" onclick="removeCriteria(this, ${iterStat.index})"></button>
+                                <button class="role-option-button kick-button" type="button" onclick="removeCriteria(this, ${iterStat.index})">
+                                    <img src="/icons/cross.svg">
+                                </button>
                             </label>
                         </div>
                     </th:block>
                 </div>
 
-                <button class="light-button" type="button" onclick="addCriteria()">Add criteria</button>
+                <div class="add-criteria">
+                    <h7>Add criteria</h7>

```

Comment:
Maybe rather "Add criterion"?

Reply:
question is will a basic user even know wtf is criterion 🙈